src/org/loklak/Caretaker.java

```diff
@@ -69,7 +68,7 @@
     public void shutdown() {
         this.shallRun = false;
         this.interrupt();
-        Log.getLog().info("catched caretaker termination signal");
+        DAO.log("catched caretaker termination signal");
```

Comment:
Checked, there is no difference in the logs:-

1)  with no source file name log :-
         before change `2017-05-12 21:03:25.051:INFO :org.loklak.data.DAO:Thread-38: got 0 suggestions for harvesting from 0 in backend`
         after change `2017-05-12 21:03:24.715:INFO :org.loklak.data.DAO:Thread-37: got 0 suggestions for harvesting from 0 in backend`

2) with source file name:-
       before change `2017-05-12 21:03:38.360:INFO :org.loklak.data.DAO:CARETAKER: Deleted 1191 outdated(hour) messages`

       after change `2017-05-12 21:03:38.360:INFO :org.loklak.data.DAO:CARETAKER: Deleted 1191 outdated(hour) messages`